manifests/olm-catalog/4.5/nfd.v4.5.0.clusterserviceversion.yaml

```diff
@@ -27,7 +27,7 @@ metadata:
         }
       ]
     description: This software enables node feature discovery for Kubernetes. It detects hardware features available on each node in a Kubernetes cluster, and advertises those features using node labels.
-    olm.skipRange: ">=4.1.0 <4.5.0"
+    olm.skipRange: ">=4.1.0 <4.6.0"
```

Comment:
What does this line do and why do we need it?

Reply:
`olm.skipRange` is the scope from which your clusterserviceversion can be upgraded from.

This is needed to enhance the OperatorHub deployment of NFD